lgc/include/lgc/patch/PatchBufferOp.h

```diff
@@ -30,6 +30,7 @@
  */
 #pragma once
 
+#include "Utils.h"
```

Comment:
Move that to the bottom to retain the include ordering

Reply:
clang-format should do that for you.